src/components/PersonList.tsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import classNames from 'classnames';
+import { NavLink, useParams } from 'react-router-dom';
+
+export const PersonList = () => {
+  const [peopleFromApi, setPeopleFromApi] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const { slug } = useParams();
+
+  const selectedPerson = slug ? slug : '';
```

Comment:
you can shorten code to
```suggestion
  const selectedPerson = slug || '';
```